app/test_helpers.go

```diff
@@ -330,6 +331,7 @@ func SignCheckDeliver(
 ) (sdk.GasInfo, *sdk.Result, error) {
 
 	tx, err := helpers.GenTx(
```

Comment:
wouldn't `time.now()` not make it non-deterministic? or are those generated txs not regenerated in simulations?

Reply:
We only use this in (nft) unit tests.